### PR TITLE
Update language-sql.md

### PR DESCRIPTION
**Pull request by imlazyone**

### CODE REVIEW

File: language-sql.md [hunk 1]
```diff
@@ -5,3 +5,10 @@
 ### Minimum Requirements
 
 MySQL keywords in uppercase.
+
+### Database table naming convention
+
+Prefix application name to table name. E.g.: `app1_users`, `app2_log_meta`.
```
Comment (xinningsu) [inline language-sql.md]: what's the application name of `juwai.com` and `agent-admin`? `jc_`, `aaa_`?

**Comment by tjoelsson** [comment]
So in the case of the lock feature, we created `ja_property_locked_changes`. This table is written to by JA and the read by XML. Does it make sense, or should we remove the ja_ prefix?

File: language-sql.md (hunk 1 above)
Comment (agirivera) [inline language-sql.md]: ? Do we have a short list of the applications and the conventional prefixes: like `juwai-admin` => `ja`, etc.?

**Comment by agirivera** [comment]
:+1:

File: language-sql.md [hunk 2]
```diff
@@ -5,3 +5,10 @@
...
+
+If the table is for multiple applications (normally this should be avoided), it's ok to leave it without the application name prefix.
```
Comment (arkhi) [inline language-sql.md]: ? Mmmm… Why should it be avoided?

Comment (imlazyone) [inline language-sql.md]: Normally we consider one database for one application, here's a good explanation: http://programmers.stackexchange.com/a/105788

Comment (arkhi) [inline language-sql.md]: I see, thanks. :) Would it be worth it to link to that URL in the README.md itself? (This is not a real question, mostly a suggestion)

**Comment by agirivera** [comment]
Still a :+1:

**Comment by imlazyone** [comment]
Wait, I think @tjoelsson had some thoughts on the table namings. Let me confirm again, do you have objections about merging this?

**Comment by tjoelsson** [comment]
I have no objections to the way it's written here.

**Pull request merged**